providers/implementations/serializers/serializer_rsa.c

```diff
@@ -109,10 +111,160 @@ int ossl_prov_print_rsa(BIO *out, RSA *rsa, int priv)
                 goto err;
         }
     }
+
+    switch (RSA_test_flags(rsa, RSA_FLAG_TYPE_MASK)) {
+    case RSA_FLAG_TYPE_RSA:
+        if (rsa_get0_pss_params_30(rsa) != NULL) {
+            if (ossl_prov_bio_printf(out, "(INVALID PSS PARAMETERS)\n") <= 0)
+                goto err;
+        }
+        break;
+    case RSA_FLAG_TYPE_RSASSAPSS:
+        {
+            const RSA_PSS_PARAMS_30 *pss_params = rsa_get0_pss_params_30(rsa);
+            int hashalg_nid = NID_sha1;
```

Comment:
I see those hardwired defaults in this file also in multiple places - so they should probably be in a header and shared

Reply:
Done